backend/app/alembic/versions/10368f38610b_fix_delete_document_error.py

```diff
@@ -19,16 +19,6 @@
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
-    op.create_table('chat',
-    sa.Column('message', sqlmodel.sql.sqltypes.AutoString(length=1024), nullable=True),
-    sa.Column('is_system', sa.Boolean(), nullable=False),
-    sa.Column('created_at', sa.DateTime(timezone=True), server_default=sa.text('now()'), nullable=False),
-    sa.Column('updated_at', sa.DateTime(), nullable=False),
-    sa.Column('id', sa.Uuid(), nullable=False),
-    sa.Column('course_id', sa.Uuid(), nullable=False),
-    sa.ForeignKeyConstraint(['course_id'], ['course.id'], ondelete='CASCADE'),
-    sa.PrimaryKeyConstraint('id')
-    )
```

Comment:
I could revert this, but I needed it to get the fresh migrations to run on my end without issues with duplicate create chat table.